### DISCUSSION TITLE
Plotter not receiving reward in wallet since may 8

### DISCUSSION THREAD

**Discussion by la-mikkamakka** [Chia-Network/chia-blockchain#15270] [Support]
Hello Guys, my rig has stopped getting rewards since May 8. I usually get 0.01 chia per day. that was half on May 8th, and nothing since. Been 4 days without any reward. Just upgraded the program from 1.6 to 1.8. Still no change. Plots are there, have node connections, I can see the transaction activity, but nothing comes my way since May 8th. 
Anyone has any idea why?

**Comment by jermie1** [comment]
Same exact thing here since May 7! What gives??
